Library/ViewModels/ActivityFriendFollowCellViewModel.swift

```diff
@@ -26,15 +26,15 @@ public final class ActivityFriendFollowCellViewModel: ActivityFriendFollowCellVi
 ActivityFriendFollowCellViewModelOutputs {
 
   public init() {
-    let friend = self.activityProperty.signal.ignoreNil()
+    let friend = self.activityProperty.signal.skipNil()
       .map(Activity.lens.user.view)
-      .ignoreNil()
+      .skipNil()
       .map(cached(friend:))
 
-    self.friendImageURL = friend.map { NSURL.init(string: $0.avatar.small) }
+    self.friendImageURL = friend.map { URL.init(string: $0.avatar.small) }
 
     self.title = friend.map {
-      let string = Strings.activity_user_name_is_now_following_you(user_name: "<b>\($0.name ?? "")</b>")
+      let string = Strings.activity_user_name_is_now_following_you(user_name: "<b>\($0.name)</b>")
```

Comment:
`$0.name` is not optional so the coalescing totally wasnt necessary. we didnt see it until now cause now swift 3 warns on unnecessary coalescing.